img2dataset/downloader.py

```diff
@@ -237,9 +247,11 @@ def download_one_file(url_list):
             with open(url_list, encoding='utf-8') as file:
                 images_to_dl = [(url, ) for url in file.readlines()]
             column_list=["url"]
-        elif input_format == "csv" or input_format=="parquet":
+        elif input_format in ["csv", "tsv", "parquet"]:
             if input_format == "csv":
                 df = pd.read_csv(url_list)
+            elif input_format == "tsv":
+                df = pd.read_csv(url_list, sep='\t')
```

Comment:
```suggestion
                df = pd.read_table(url_list)
```

Reply:
I think it's the exact same. Let me know which one you prefer.

Reply:
read_table.

Reply:
Why ? Read csv seems more explicit